packages/custom-url/src/Infrastructure/Symfony/Normalizer/CustomUrlNormalizer.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\CustomUrl\Infrastructure\Symfony\Normalizer;
```

Comment:
```suggestion
namespace Sulu\CustomUrl\Infrastructure\Symfony\Serializer;
```

this is connection point to symfony/serializer package